### PR TITLE
vote for zyguan as maintainer

### PR DESCRIPTION
**Pull request by cfzjywxk**

### CODE REVIEW

**Review by zz-jason — APPROVED** [review]
LGTM

**Review by disksing — APPROVED** [review]

**Review by bb7133 — APPROVED** [review]
LGTM

File: teams/tidb/membership.json [hunk 1]
```diff
@@ -28,6 +28,7 @@
         "winoros",
         "wjhuang2016",
         "zimulala",
+	"zyguan",
```
Comment (bb7133) [inline teams/tidb/membership.json:31]: Need to remove `zyguan` from 'committers'.

**Review by bb7133 — CHANGES_REQUESTED** [review]

**Review by bb7133 — APPROVED** [review]
LGTM

**Pull request merged**